package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "k6-es6",
```

Comment:
Please don't forget to fill this (and other properties like `version`, `description` and so on) with a proper values